hathor/p2p/node_sync.py

```diff
@@ -248,22 +249,26 @@ def get_cmd_dict(self) -> Dict[ProtocolMessages, Callable[[str], None]]:
             ProtocolMessages.NOT_FOUND: self.handle_not_found,
         }
 
+    @property
```

Comment:
I don't like to use properties. Is it ok to remove it?

Reply:
Leave it as a method? OK